rosa_architecture/rosa-sts-about-iam-resources.adoc

```diff
@@ -85,6 +85,17 @@ include::modules/rosa-sts-about-operator-role-prefixes.adoc[leveloffset=+2]
 
 * For steps to create the cluster-specific Operator IAM roles using a custom prefix, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-sts-creating-cluster-customizations-cli_rosa-sts-creating-a-cluster-with-customizations[Creating a cluster with customizations using the CLI] or xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-sts-creating-cluster-customizations-ocm_rosa-sts-creating-a-cluster-with-customizations[Creating a cluster with customizations by using {cluster-manager}].
 
-include::modules/rosa-sts-oidc-provider.adoc[leveloffset=+1]
+[id="rosa-sts-oidc-provider-requirements-for-operators_{context}"]
+== Open ID Connect (OIDC) requirements for Operator authentication
+
+For ROSA installations that use STS, you must create a cluster-specific OIDC provider that is used by the cluster Operators to authenticate or create your own OIDC configuration for your own ODIC provider.
```

Comment:
I didn't quite got this

Is the cluster-specific OIDC provider the classic OIDC configuration? I'm not sure how we are going to name it, but in my mind I call it unregistered managed oidc configuration, which is definitely not something understandable for and end customer

Reply:
@gdbranco For the time being, I am referring to OIDC provider as the classic OIDC configuration. We still have the `rosa create oidc-provider` command in the CLI(1) and in the docs (2). We could revise this entire section and explain the terms better if you think that would help.


----
1. 
![image](https://github.com/openshift/openshift-docs/assets/16167833/6cc5e7b9-53be-4870-bce0-ef69fd245061)

2. 
![image](https://github.com/openshift/openshift-docs/assets/16167833/25172180-0565-4dea-81a5-092b4167daf8)